### ISSUE TITLE
Replace github submodules by cocoapods test dependency

### ISSUE THREAD

**Issue by corinnekrych** [AliSoftware/OHHTTPStubs#86]
Although not an issue in itself, github submodules work when you don't forget to init them :))
Why not add a Podfile with a Test dependency on AFNetworking? 
wdyt @AliSoftware ?

**Comment by AliSoftware** [comment]
At the time I created `OHHTTPStubs`, `target xxx do` directive was not part of the CocoaPods DSL.

That that it is, I see no objection to migrating to `target 'UnitTest' { pod 'AFNetworking' }` instead of a submodule :+1:

If we do that, don't forget to add the README ([those lines](https://github.com/AliSoftware/OHHTTPStubs/commit/5c5e5faf27e58c0be830ba735b65d9ae0df92b15#diff-04c6e90faac2675aa89e2176d2eec7d8R203))

**Comment by AliSoftware** [comment]
This future config has been introduced in the README [here](https://github.com/AliSoftware/OHHTTPStubs/commit/02cc9fcc0cdfca79e1968df9e3e072c256e5e593#diff-04c6e90faac2675aa89e2176d2eec7d8R127) and we should get rid of the whole paragraph once this is done

**Comment by corinnekrych** [comment]
Great I'm working on a PR.

**Comment by AliSoftware** [comment]
Cool thx!

**Comment by AliSoftware** [comment]
This has been fixed by #90 :+1:  Thx
